src/test/java/com/booster/server/NoteControllerTest.java

```diff
@@ -113,4 +113,38 @@ void returnsBatchOfNotesWithRespectToLastSeenAt() throws Exception {
                 .andExpect(jsonPath("$.[1].content").value(firstNoteContent));
     }
 
+    @Test
+    void deletesById() throws Exception {
+        Long id1 = noteRepository.save(new NoteEntity()
+                        .setContent(firstNoteContent))
+                .getId();
+        Long id2 = noteRepository.save(new NoteEntity()
+                        .setContent(secondNoteContent))
+                .getId();
+
+        mockMvc.perform(MockMvcRequestBuilders.delete("/note/{id}", id1))
+                .andDo(print())
+                .andExpect(status().isNoContent());
+
+        assertThat(noteRepository.findById(id1)).isEmpty();
+        assertThat(noteRepository.findById(id2)).isNotEmpty();
+    }
+
+    @Test
+    void failsToDeleteById() throws Exception {
+        Long id1 = noteRepository.save(new NoteEntity()
+                        .setContent(firstNoteContent))
+                .getId();
```

Comment:
Do we need this successfully saved note if we test the fail scenario?

Reply:
That's a good question.

I added it to ensure that the existing notes are not affected by a failing call.